network/transport/v2/conversation.go

```diff
@@ -159,6 +159,19 @@ func (envelope *Envelope_TransactionListQuery) conversationID() []byte {
 	return envelope.TransactionListQuery.ConversationID
 }
 
+func (envelope *Envelope_TransactionRangeQuery) setConversationID(cid conversationID) {
+	envelope.TransactionRangeQuery.ConversationID = cid.slice()
+}
+
+func (envelope *Envelope_TransactionRangeQuery) conversationID() []byte {
+	return envelope.TransactionRangeQuery.ConversationID
+}
+
+func (envelope *Envelope_TransactionRangeQuery) checkResponse(other isEnvelope_Message) error {
+	// TODO: Not sure what to do here
```

Comment:
according to spec every TX received in the TransactionList must have a LC value in the requested range.